_posts/2016-12-06-deprecating-nodes.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Deprecating Nodes
+author: David Davis
+tags:
+  - release
+  - platform
+  - 3.0
+---
+
+Nodes were introduced in Pulp in 2.1.0 and provided a way to keep multiple Pulp
+servers in sync. However, maintaining Nodes in Pulp meant having to maintain a
+separate codebase and given that much of the functionality of Nodes can be
+acheived by running Pulp straight out of the box, we've decided to remove Nodes
+in our next major release of Pulp 3.0. With Pulp 2.11, Nodes will be officially
+deprecated.
```

Comment:
I think there is more to be said than the maintenance effort. What about this for the first paragraph which includes the above comments:

Nodes were introduced in Pulp in 2.1.0 and provided a way to keep multiple Pulp servers synchronized. However, Nodes uses separate codepaths from the normal sync+publish codepaths which creates at a few problems:

* Nodes is less tested than sync+publish even though it provides a similar feature set. This greatly reduces the user to user usage and testing benefit.
* Fixing bugs in sync+publish does not fix bugs in Nodes and vice-versa
* Nodes has a good amount of bugs already reported that are still unfixed

Given that much of the functionality of Nodes can be achieved by running Pulp with sync+publish workflow, the core team is removing Nodes in the next major release of Pulp 3.0. With Pulp 2.12, Nodes will be officially deprecated.